### DISCUSSION TITLE
Subscriptions page in console - colors (Office hours: Oct 1, 2020)

### DISCUSSION THREAD

**Discussion by ksabniskrishna** [twilio-labs/paste#800] [Office hours] [answered]
The specific questions I have are...

- Is there a success alert? 
- Am I using Illustrations correctly?  
- Are the cards styled okay? 
- Am I using modal correctly?  

Background: We're planning on starting to build MLP for subscriptions. Followed the illustration guidelines, want to do a one last check if I am using the modal and cards right. 
Stage of design:Prototype
Links to mockups or prototype: https://twilio.invisionapp.com/share/C5YWFLKMDKY

**Comment by kate-porter** [comment]
Hey @ksabniskrishna! Overall, the design is awesome, and the component usage looks good. I added some comments and questions in Invision.

**Reply by ksabniskrishna** [reply]
Hey folks! Thanks! I fixed a few and we can talk about others in the office hours! Thanks!

**Comment by richbachman** [comment]
Hi @ksabniskrishna 

It looks like the InVision prototype has been archived. Will you share a new link or unarchive the previous one?

Without seeing the prototype, I'm not sure but would the success Toast work in your use case? 
https://paste.twilio.design/components/toast#success-toast

**Comment by ksabniskrishna** [comment, marked as answer]
Hey folks! Apologies for the late notice - but I have a project update: We're putting this project on hold due to some missing exec partners on pricing and billing teams. I will bring this back when its back on track. 

@richbachman thanks for the suggestion. I was using the toast. 
Here's the new link: 
 https://twilio.invisionapp.com/share/PNYV98JBA2Q password: twilio123
